src/components/CompareResults/RunValues.tsx

```diff
@@ -42,8 +42,7 @@ function RunValues(props: RunValuesProps) {
     <div className={styles.container}>
       {application ? (
         <div>
-          <b>{name}:</b> {formatNumber(median)} {measurementUnit} ({application}
-          )
+          <b>{name}:</b> {formatNumber(avg)} {measurementUnit} ({application})
```

Comment:
From what I can see here, looks like all the necessary values have been formatted. This looks ready to merge. What do you say @julienw